assets/src/components/editing/editor/handlers/paste.ts

```diff
@@ -0,0 +1,21 @@
+import { Transforms, Editor as SlateEditor } from 'slate';
+import * as ContentModel from 'data/content/model';
+
+export const onPaste = (editor: SlateEditor, e: React.ClipboardEvent<HTMLDivElement>) => {
+  if (!e.clipboardData) {
+    return;
+  }
+
+  // The clipboard item 'type' attr is a mime-type. look for image/xxx.
+  // 'Rich' images e.g. from google docs do not work.
+  const images = [...e.clipboardData.items].filter(({ type }) => type.includes('image/'));
+  if (images.length === 0) {
+    return;
+  }
+
+  return images
+    .map(image => image.getAsFile())
+    .map(URL.createObjectURL)
```

Comment:
This doesn't seem to be working as you intended.  Here is what I tried:

1. I copied into my clipboard a file that I have on my local machine (from Finder, selected the file and right-clicked "Copy")
2. I pasted into the content of a page, the image rendered correctly
3. I left the page and returned and I see a "broken image" placeholder.  

It looks like this is what it serialized in the content:

```%{"caption" => "x", "children" => [%{"text" => ""}], "display" => "block", "id" => "3200919815", "src" => "blob:https://localhost/cfb87685-f843-49a3-8ae9-ed3fa185e45b", "type" => "img"}```

Reply:
For this to work correctly, I believe a step will need to be added that uploads the pasted image into the media library, and then puts that URL in for the "src"